Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[workspace]
+
+# this is for the new crate structure. The legacy code (current CML) still resides in the `rust` directory.
+members = [
+    "core",
```

Comment:
We'll still need to figure out what to do with the JSON schema generating stuff as we currently have multiple in the project now with CIP25.

Reply:
could you share a little more context please? what's the problem? seems like I don't have full picture